binary_search.py

```diff
@@ -0,0 +1,24 @@
+def binary_search(my_list,item):
+  '''
```

Comment:
'''
Search item in the sorted list.
INPUT:
    my_list : list of items
    item: item to be searched
OUTPUT:
    mid : index of search item 
'''